Demos/01-create-app/graph-tutorial/package.json

```diff
@@ -6,15 +6,15 @@
     "start": "node ./bin/www"
   },
   "dependencies": {
-    "@microsoft/microsoft-graph-client": "^1.5.2",
+    "@microsoft/microsoft-graph-client": "^1.6.0",
```

Comment:
create-app.md still has the old versions, can you update that to match?

Reply:
Thanks for updating, but there's also dotenv and express-session that still specify the old versions in create-app.md.

Reply:
Fixed in the last commit...